composer.json

```diff
@@ -1,41 +1,46 @@
 {
-    "name": "level-level/clarkson-core",
-    "type": "wordpress-muplugin",
-    "license": "GPL-2.0-or-later",
-    "description": "A plugin to write Object-Oriented code in combination with the Twig templating engine while keeping the WordPress Way of working in mind.",
-    "authors": [
-        {
-            "name": "Level Level",
-            "homepage": "https://level-level.com"
-        },
-        {
-            "name": "Niels de Blaauw"
-        },
-        {
-            "name": "Jaime Martinez"
-        },
-        {
-            "name": "Robin Aldenhoven"
-        }
-    ],
-    "autoload": {
-      "classmap": ["wordpress-objects/", "lib/"]
-    },
-    "scripts": {
-        "fix" : [
+	"name": "level-level/clarkson-core",
+	"type": "wordpress-muplugin",
+	"license": "GPL-2.0-or-later",
+	"description": "A plugin to write Object-Oriented code in combination with the Twig templating engine while keeping the WordPress Way of working in mind.",
+	"authors": [
+		{
+			"name": "Level Level",
+			"homepage": "https://level-level.com"
+		},
+		{
+			"name": "Niels de Blaauw"
+		},
+		{
+			"name": "Jaime Martinez"
+		},
+		{
+			"name": "Robin Aldenhoven"
+		}
+	],
+	"autoload": {
+	  "classmap": ["wordpress-objects/", "lib/"]
+	},
+	"scripts": {
+		"fix" : [
 			"phpcbf"
 		],
 		"test": [
 			"composer validate",
-			"phpcs"
+			"phpcs",
+			"@phpunit"
+		],
+		"phpunit":[
+			"phpunit"
 		]
-    },
-    "require" : {
-        "twig/twig":"^1.2",
-        "twig/extensions":"^1.3"
-    },
-    "require-dev": {
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
-        "wp-coding-standards/wpcs": "^2.1"
-    }
+	},
+	"require" : {
+		"twig/twig":"^1.2",
+		"twig/extensions":"^1.3"
+	},
+	"require-dev": {
+		"dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
+		"wp-coding-standards/wpcs": "^2.1",
+		"10up/wp_mock": "^0.4.2"
```

Comment:
these packages require php7.1 to be installed. We should define that in travis and composer json i think?